### ISSUE TITLE
 Resnet20 based on adder_cuda seems to have difficulty converging

### ISSUE THREAD

**Issue by 154115081020** [LingYeAI/AdderNetCUDA#4]
I try to train resnet20 for classification task on cifar10 dataset. But when using adder_cuda, the network seems to have difficulty converging. So, I am curious about the author's experimental results on the cifar10 dataset.

**Comment by LingYeAI** [comment]
Have you ever tried different learning rate?
Reduce the learning rate try it again, train for 400 epochs and see what happens.
